api/.gitignore

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
I think it is cleaner to have a single `.gitignore` in the project (in the root directory)

Reply:
Done! .gitignore is now in root repository.

Reply:
Well, this file is still here. And we have two gitignore files now.

I guess that this file is created by `composer install`, and so as you are running it from inside api/ folder, it creates a new one if it nees to add something. Maybe the best will be to get the file back to this folder to make composer happy.

Reply:
Ping. This is still here.